app/controllers/users_controller.rb

```diff
@@ -1,10 +1,23 @@
 class UsersController < ApplicationController
+  load_and_authorize_resource
 
   # Information about the user corresponding to the ID in the URI
   #
   # === Variables
   # - user: the user with the given ID
   # - owned_methods: methods owned by this user
+  def index
```

Comment:
Remove index, changeAdmin, changeEditor, and changeBasic methods here, now only used in AdministratorController